lld/test/ELF/aarch64-memtag-startstop.s

```diff
@@ -0,0 +1,48 @@
+# REQUIRES: aarch64
```

Comment:
This is an extra test for the basic test `aarch64-memtag-globals.s`, so the name can be `aarch64-memtag-globals-startstop.s`